arch/arm/src/nrf52/nrf52_lowputc.c

```diff
@@ -216,7 +216,7 @@ void nrf52_usart_configure(uintptr_t base, const struct uart_config_s *config)
  ****************************************************************************/
 
 #ifdef HAVE_UART_DEVICE
-void nrf52_usart_disable(uintptr_t base)
+void nrf52_usart_disable(uintptr_t base, const struct uart_config_s *config)
```

Comment:
Mateuz, I think it is a good idea to add FAR to all these "const struct uart_config_s *", what do you think?

Reply:
Yes, you are right. Fixed